supabase/migrations/20260816201646_role_bindings_require_org_membership.sql

```
@@ -0,0 +1,207 @@
+          FROM public.role_bindings AS owner_membership
+          WHERE owner_membership.principal_type = public.rbac_principal_user()
+            AND owner_membership.principal_id = apikeys.user_id
+            AND owner_membership.org_id = p_org_id
+            AND (owner_membership.expires_at IS NULL OR owner_membership.expires_at > now())
+        )
+    )
+    ELSE false
+  END
+$$;
+
+ALTER FUNCTION public.role_binding_principal_allowed_for_org(text, uuid, uuid, text) OWNER TO postgres;
```

SQLFluff (4.2.2)

[error] Line is too long (103 > 80).

(LT05)